README.md

```diff
@@ -4,36 +4,28 @@
 ![Tensorflow](https://img.shields.io/badge/tensorflow-v2.9.0+-success.svg)
 [![Contributions Welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/keras-team/keras-cv/issues)
 
-# Vision
-A computer vision library dedicated for auto-driving, robotics and on device applications.
 
 # Mission
 
-KerasCV is a layered repository consisting of core components and modeling components.
+KerasCV is a computer vision library of modular computer vision oriented Keras components.
```

Comment:
"computer vision" appears twice in this sentence and that seems redundant. Maybe drop the first one?